packages/arb-token-bridge-ui/src/state/app/utils.ts

```diff
@@ -283,11 +283,11 @@ export const getStandardizedTimestamp = (date: string | BigNumber) => {
   return date.toNumber()
 }
 
-export const getStandardizedTime = (standatdisedTimestamp: string) => {
+export const getStandardizedTime = (standatdisedTimestamp: number) => {
```

Comment:
Let's fix the spelling of `standatdisedTimestamp` -> `standardizedTimestamp` for all instances in this file.

Reply:
updated